src/python/WMCore/Credential/Proxy.py

```diff
@@ -628,11 +635,18 @@ def vomsExtensionRenewal(self, proxy, voAttribute='cms'):
             self.logger.error("Error while checking retrieved proxy timeleft for %s" % proxy)
             return
 
+        # timeLeft indicates how maby seconds the proxy is still valid for
```

Comment:
there are a few typos in these comments, can you please fix them.

Reply:
OK. Also moved some lines around so comments are close to code they refer to.